tree-sitter-go/tree-sitter-go.cabal

```diff
@@ -1,21 +1,21 @@
 cabal-version:       2.2
 name:                tree-sitter-go
-version:             0.2.0.0
+version:             0.2.0.1
 synopsis:            Tree-sitter grammar/parser for Go
 description:         This package provides a parser for Go suitable for use with the tree-sitter package.
+license:             BSD-3-Clause
 homepage:            https://github.com/tree-sitter/tree-sitter-go#readme
 author:              Max Brunsfeld, Tim Clem, Rob Rix, Josh Vera, Rick Winfrey, Ayman Nadeem, Patrick Thomson
 maintainer:          tclem@github.com
 copyright:           2017 GitHub
 category:            Tree-sitter, Go
-license:             BSD-3-Clause
 build-type:          Simple
 
 library
   exposed-modules:     TreeSitter.Go
-  build-depends:       base                 >= 4.12 && < 5
-                     , tree-sitter          == 0.3.0.0
-                     , template-haskell     >= 2.12.0.0 && < 2.15.0.0
+  build-depends:       base              >= 4.12 && < 5
+                     , tree-sitter       >= 0.3
```

Comment:
Likewise, we really don’t want to have to keep doing this; `tree-sitter-go` itself is in fact dependent only on a few small details of `tree-sitter` (e.g. the `Language` type) and not on the generated ASTs.

The same goes for `tree-sitter-python` (which actually generates ASTs), because it will compile correctly with either 0.3 or 0.4.